editions/tw5.com/tiddlers/wikitext/HTML in WikiText.tid

```diff
@@ -14,7 +16,36 @@ This is my nice and simple block of text. HelloThere
 </article>
 ```
 
-[[Widgets share the same syntax as HTML tags|Widgets in WikiText]], and so the following information applies to them, too.
+!! Comments
+
+A html-comment as described above has been possible since ~TiddlyWiki Version 5.1.0.
+
+<<.from-version 5.2.0>> It will be possible to use "performant" ''html-comments together with pragmas''. Comments as described below can be cached by the TW core, so they are performant.
```

Comment:
We don't refer to TiddlyWiki as TW in the docs. This might be better expressed as:

```
 <<.from-version 5.2.0>> Comments can now be freely intermixed with pragmas as well as within the main body of a block of wikitext.
```